docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -0,0 +1,247 @@
+
+   b. Give the new account a name that clearly marks it as app-specific; the idea is to differentiate it from the merchant-specific payment and subscription management account. This guide uses **BigCommerce App**.
+   
+   ![BigCommerce App](https://storage.googleapis.com/bigcommerce-production-dev-center/images/BigCommerce-app-image.png "The Stripe dashboard's new account dropdown menu")
+   
+   c. Under **Get started with Stripe**, copy the **Secret key**. In a later step, you will use the secret key to update the `STRIPE_SECRET_KEY` environment variable in the .env file.
+
+### Enable Stripe Connect for Platforms
+
+**During this instruction sequence, make sure that the left side of your Stripe Dashboard's top menu bar indicates that you're in the app-specific account.**
+
+<!-- theme: info -->
+> #### Test mode
+> You can simulate transactions in test mode to confirm your integration works correctly; the following steps are configured for test mode.
+
+1. Click the **Connect** button to enable [Stripe Connect for Platforms](https://dashboard.stripe.com/test/connect/accounts/overview). 
+
+2. Click **Get started**.
+
+3. Select **Platform or Marketplace**, then click the **Continue** button at the lower right.
+
+4. To configure the app-specific account's Stripe Connect settings, go to [**Settings > Connect settings**](https://dashboard.stripe.com/test/settings/connect). Under **Integration** perform the following steps:
+   
+   a. Copy the **Test mode client ID**. In a later step, you will use the client ID to update `NEXT_PUBLIC_STRIPE_CLIENT_ID` in the .env file.
+
+   b. Set **OAuth settings** to **OAuth for Standard accounts**. 
+
+   c. Add **Redirects**. Click the **+ Add URI** button and add your app's callback URI.
+   
+   ```http title="Your app's Stripe callback URI"
+   https://{ngrok_id}.ngrok.io/stripe/callback
+   ```
+
+## Configure the BigCommerce store
+
+To configure the store to make subscription charges, take the following steps:
+
+   - In the [Stripe settings section](https://login.bigcommerce.com/deep-links/settings/payment/stripev3) of the BigCommerce store control panel, make sure **Test Mode** is set to **Yes**.
+   
+   ![stripe-settings](https://storage.googleapis.com/bigcommerce-production-dev-center/images/stripe-settings.png)
+   
+   - To start a subscription, shoppers must check out using a stored card. To turn on that functionality, go to [Stripe settings](https://login.bigcommerce.com/deep-links/settings/payment/stripev3) in the BigCommerce store control panel, find the **Stored Credit Cards** section, and toggle on **Enable stored credit cards with Stripe**. 
+   
+   ![stored-credit-cards](https://storage.googleapis.com/bigcommerce-production-dev-center/images/stored-credit-cards.png)
+   
+   - To save a card during the transaction, sign in to the BigCommerce store as a shopper, or create a new account during checkout. In the payments step, select **Save this card for later**. 
+
```

Comment:
consider providing at least one screenshot if possible. Also, I recommend using "credit card" instead of just "card" if the UI says the same.